VersionHistory.md

```diff
@@ -1,5 +1,11 @@
 # Version History
 
+## 3.2.0
+
+* `SQLiteLog` only calls `sqlite3_config` to initialize logging when an event handler is attached to `SQLiteLog.Log`.
+  * It is no longer called automatically from the `SQLiteConnection` constructor. This fixes a `TypeInitializationException` and `AppDomainUnloadedException` in `SQLiteLog`.
+  * If you are using `SQLiteLog.Log`, you _must_ attach an event handler before calling any other methods of this library. This is a [limitation of SQLite](https://www.sqlite.org/c3ref/config.html).
```

Comment:
What happens if you don't? Logging doesn't work? The app crashes?

Reply:
It doesn't work. `sqlite3_config` returns `SQLITE_MISUSE` IIRC.

Reply:
Added another bullet point to the release notes.